terraform/variables.tf

```diff
@@ -95,3 +95,13 @@ variable "rollbar_disabled" {
   description = "Flag to turn off rollbar reporting"
   default     = "false"
 }
+
+variable "custom_domain" {
+  type        = string
+  description = "Name of custom domain created in the cf org"
+}
+
+variable "custom_hostname" {
+  type = string
```

Comment:
@Stretch96 I think the linting wants this to be aligned with the row under it, if I understand its output right.

I can't review the rest, but this might unblock the CI running successfully.

Reply:
Ah yes, I forgot to run `terraform fmt`, thanks 👍